### ISSUE TITLE
Option to disable "Consider following"

### ISSUE THREAD

**Issue by heini** [keybase/client#11487]
Hi,

can we please have an option to disable this? It's just annoying. Or, even better, have it disabled by default with an option to enable it, for those who care :smile:

**Comment by itsgreggreg** [comment]
Agreed this feature is obnoxious, irrelevant and confusing at best and doesn't belong in an application centered on identity verification and trust.

**Comment by heini** [comment]
Damn, missed its first aniversary! *SCNR*

**Comment by drewbug** [comment]
appears in the code as `followSuggestions`

**Comment by schrodyn** [comment]
Any movement on this?

**Comment by snugghash** [comment]
Bit of a hijack, but I'd like to understand whether this is a risk beyond just a UI annoyance. How are follow suggestions computed? As far as my [perusal](https://github.com/keybase/client/blob/38a96307aa34f4ebafff79aa45a7c6245dd2ae38/shared/actions/people.tsx#L110) of the codebase goes, it seems like it's generated (closed-source) server-side and we have no idea how it happens. 

Is it safe to assume all of our metadata (including whether we ever "looked up" someone from the keybase servers) is used to generate these suggestions? What about other people - if someone else looks us up, can we assume that increases the chance of their profile showing up as a follow suggestion? Is this eventually going to turn into facebook now?

**Comment by heini** [comment]
@snugghash, given the age of this request (and some others), I've given up any hope that anything will happen...

**Comment by heini** [comment]
My experience is that the useful issues are simply ignored, like this one, or the one about translating the UI texts. So I've given up on this (see above) and setup my own Matrix home-server...

**Comment by mathieujobin** [comment]
I also use Matrix, and its not about to replace Keybase for me ...
